topics/fair/tutorials/earth_system_rocrate/slides.html

```diff
@@ -0,0 +1,135 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "An earth-system RO-Crate"
+video: true
+questions:
+  - "How to generate a RO-Crate"
+  - "How to follow a Marine Omics study"
+  - "How to read a RO-Crate ?"
+objectives:
+  - "Understand RO-Crate in an earth-system context"
+time_estimation: 1H
+tags:
+  - earth-system
+  - ro-crate
+  - ocean
+  - marine omics
+key_points:
+  - "RO-Crate in earth-system analysis"
+  - "Marine omics workflow "
+contributors:
+  - Marie59
+
+---
+
+# Workflow context
+
+The workflow used as an example here goes from :
+- First, produce a protein FASTA file from a nucleotide FASTA file using Prodigal (a tool that predicts protein-coding genes from DNA sequences).
+- Then, use InterProscan to create a tabular. Interproscan is a batch tool to query the InterPro database. It helps identify and predict the functions of proteins by comparing them to known databases.
+- Finally, discover SanntiS both to build genbank and especially to detect and annotate biosynthetic gene clusters (BGCs).
+---
+
+## The workflow itself
+
+![Marine Omics identifying biosynthetic gene clusters worflow image](./images/workflow_marino.png)
+---
+
+## Run the workflow
+
+Import and run the following workflow https://earth-system.usegalaxy.eu/u/marie.josse/w/marine-omics-identifying-biosynthetic-gene-clusters
```

Comment:
You can now embedd workflows. Have yout tried if this is working in slides?

https://galaxyproject.org/news/2024-04-26-workflows-workflows-workflows/